src/logging.h

```diff
@@ -1,26 +1,20 @@
-#ifndef _CH8_SRC_DEBUG_H_
-#define _CH8_SRC_DEBUG_H_
+#ifndef LOGGING_H
+#define LOGGING_H
 
 #include <stdio.h>
 
 #include "cli.h"
 
-typedef enum log_level {
-    DEBUG,
-    INFO,
-    WARNING,
-    ERROR,
-    FATAL
-} log_level;
+typedef enum log_level { DEBUG, INFO, WARNING, ERROR, FATAL } log_level;
```

Comment:
Personally, having all enum values beneath each other is more readable to me

Reply:
Agreed, I didn't notice that. I'll look into it.

Reply:
`AllowShortEnumsOnASingleLine: false`

should do the trick

Reply:
It does the trick!